tests/documentation/test_files/DocumentationStyleBear/bad_file4.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def myfunc(a, b, x):
+    """
+
```

Comment:
No haven't seen it either that someone leaves an empty line^^
So yes can be easily added :+1: